apps/web/vibes/soul/examples/sections/header-section/electric.tsx

```diff
@@ -4,6 +4,14 @@ import { logo, navigationLinks } from '@/vibes/soul/examples/primitives/navigati
 import { HeaderSection } from '@/vibes/soul/sections/header-section'
 
 export default function Preview() {
+  const linksPromise = new Promise<{ href: string; label: string }[]>(res =>
```

Comment:
I may not have full context but why are we adding hard coded delays here? Obviously, we have streaming if loading the data takes some amount of time, but would we manually add the delay?